tests/test_core.py

```diff
@@ -135,3 +135,16 @@ def test_simple_record_from_xml():
     expected = {'main_entry_personal_name': {'personal_name': 'Donges, Jonathan F'}}
 
     assert data == expected
+
+
+def test_none_value():
+    """Test none value."""
+    overdo = dojson.Overdo()
+
+    @overdo.over('024', '^024..')
+    def match(self, key, value):
+        return None
+
+    data = overdo.do({'0247_': 'this should not be added'})
+
+    assert data.get('024', "not exists") == "not exists"
```

Comment:
What about being more explicit `assert '024' not in data, '024 should not be added to data'`?